generator/standalone_main.cc

```diff
@@ -0,0 +1,132 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/internal/absl_str_join_quiet.h"
+#include "google/cloud/log.h"
+#include "google/cloud/status_or.h"
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+#include "generator/generator.h"
+#include "generator/generator_config.pb.h"
+#include <google/protobuf/compiler/command_line_interface.h>
+#include <google/protobuf/text_format.h>
+#include <fstream>
+#include <future>
+#include <iostream>
+#include <sstream>
+
+ABSL_FLAG(std::string, config_file, "./generator/generator_config.textproto",
```

Comment:
These relative path defaults imply that this binary should be run from a specific directory so that the default paths work. I recommend avoiding this issue by omitting this default path value, and the user can just specify `--config_file` with the appropriate relative path from *their* cwd.